ansible/playbooks/tls/install-certs.yml

```diff
@@ -0,0 +1,58 @@
+- hosts: redpanda
+
+  tasks:
+  - name: Copy CA Certs
+    copy:
+      src: ca/ca.crt
+      dest: /etc/redpanda/certs/truststore.pem
+      owner: redpanda
+      group: redpanda
+
+  - name: Copy Node Certs
+    copy:
+      src: certs/{{ansible_hostname}}/node.crt
+      dest: /etc/redpanda/certs/node.crt
+      owner: redpanda
+      group: redpanda
+
+  - name: Configure via RPK
+    shell:
+      cmd: |
+        rpk redpanda config set redpanda "
+
+          kafka_api:
```

Comment:
🔥 

this is awesome. @0x5d  - check it out :D

Reply:
Great! 😊